eng/pipelines/templates/stages/python-analyze-weekly-standalone.yml

```diff
@@ -93,6 +93,23 @@ extends:
                   env:
                     GH_TOKEN: $(azuresdk-github-pat)
 
+                - script: |
```

Comment:
Not directly about this PR, but @scbedd any idea why these two templates are basically duplicated (just one has 1es extends) vs. the standalone calling into the non-standalone? I can throw up a fix for this if there's no specific reason/divergence.

Reply:
No I think it's more of an 

> I never reviewed this template usage

I'll throw up a fix you don't need to do any work here.

Reply:
I am not certain if this is the reason, but we point to the standalone template for libraries that don't have live tests, but where we still want to run analyze weekly: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ai/tests.yml

But if we could just point to one template in cases of (live tests + analyze weekly) or (just analyze weekly) that would be ideal.